securitycenter/snippets/snippets_findings.py

```diff
@@ -564,8 +584,12 @@ def group_findings_and_changes(source_name):
     # source_name is the resource path for a source that has been
```

Comment:
```suggestion
    # 'source_name' is the resource path for a source that has been
```
I believe similar changes are required above.